pytest.ini

```diff
@@ -0,0 +1,3 @@
+[pytest]
+testpaths = test
```

Comment:
No, we do not. But since you use this fancy trick to get netlib to test on py3 I'm not sure how to handle it...